src/pages/proposals/ProposalDetail.vue

```diff
@@ -645,7 +647,8 @@ export default {
       .bottom-rounded.shadow-up-7.fixed-bottom.z-top(v-if="$q.screen.lt.lg")
         voting(v-if="proposalParsing.status(proposal) !== 'drafted'" :proposal="proposal" :title="null" fixed)
 .proposal-detail.full-width(v-else-if="$q.platform.is.desktop")
-  .row(v-if="!$apollo.queries.proposal") Loading...
+  div(v-if="$apollo.queries.proposal.loading" class="row justify-center q-my-md")
```

Comment:
I suggest you add computed value such as `loading` and for now return just `this.$apollo.queries.proposal.loading` as this page has many moving elements and we can extend other condition for loading easily